jsonrpc/types.go

```diff
@@ -245,7 +245,7 @@ func toHeader(h *types.Header) *header {
 	return res
 }
 
-type receipt struct {
+type Receipt struct {
```

Comment:
Do not expose this struct, it is only used to serialize data when sending it as a request result. Just use the `ethgo.Receipt` when deserializing in `EthClient`.